jaydebeapi/__init__.py

```diff
@@ -640,12 +640,26 @@ def to_py(rs, col):
         return getattr(java_val, java_method)()
     return to_py
 
+def _java_to_py_bigdecimal(java_method):
+    def to_py(rs, col):
+        java_val = rs.getObject(col)
+        if java_val is None:
+            return
+        scale = getattr(java_val, 'scale')()
```

Comment:
I think it is safer if you have a sensible default. I don't know whether all JDBC drivers have the `scale` attribute.
```suggestion
        scale = getattr(java_val, 'scale', None)()
```

Reply:
Instead of adding default, it is better check whether 'scale' attribute exists, because None cannot be invoked as a method.